bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -136,6 +144,14 @@ spec:
       - name: host-sys
         hostPath:
           path: /sys
+      - name: host-run-systemd
+        hostPath:
+          path: /run/systemd
+          type: Directory
+      - name: host-var-log-openvswitch
+        hostPath:
+          path: /var/log/openvswitch
+          type: Directory
```

Comment:
When OVS is *not* started on the host, what creates this Directory?   The container will fail mount constraints until the host creates ```/var/log/openvwitch```

Reply:
Good point, though I don't think it's possible to get in that state. I'll update it.